addon/helpers/without.js

```diff
@@ -16,7 +15,7 @@ export function without(needle, haystack) {
     return false;
   }
 
-  if (typeOf(needle) === 'array' && get(needle, 'length')) {
+  if (isEmberArray(needle) && get(needle, 'length')) {
```

Comment:
`isEmberArray` is a shim for [`Ember.isArray`](https://www.emberjs.com/api/ember/2.14/namespaces/Ember/methods/isArray?anchor=isArray) which will return `true` for both native arrays and array-like objects.